aws-lc-fips-sys/builder/main.rs

```diff
@@ -130,39 +130,42 @@ fn prefix_string() -> String {
     format!("aws_lc_fips_{}", VERSION.to_string().replace('.', "_"))
 }
 
-#[cfg(any(
-    feature = "bindgen",
-    not(all(
-        any(target_arch = "x86_64", target_arch = "aarch64"),
-        any(target_os = "linux", target_os = "macos"),
-        any(target_env = "gnu", target_env = "musl", target_env = "")
-    ))
-))]
+#[cfg(feature = "bindgen")]
 fn target_platform_prefix(name: &str) -> String {
     format!("{}_{}", target().replace('-', "_"), name)
 }
 
 pub(crate) struct TestCommandResult {
-    output: Box<str>,
-    status: bool,
+    #[allow(dead_code)]
+    stderr: Box<str>,
+    #[allow(dead_code)]
+    stdout: Box<str>,
     executed: bool,
+    status: bool,
 }
 
 fn test_command(executable: &OsStr, args: &[&OsStr]) -> TestCommandResult {
-    if let Ok(result) = Command::new(executable).args(args).output() {
-        let output = String::from_utf8(result.stdout)
+    if let Ok(mut result) = Command::new(executable).args(args).output() {
+        result.stderr.truncate(4112);
```

Comment:
why 4112?

Reply:
No reason in particular. It was 2112, then thought that might not be enough. I will make it a named constant to remove the "magic" from it.